File: benchmarks/fibonacci/fib.go
package main

import "fmt"

func fibonacci(n uint32) uint32 {
	if n <= 1 {
		return n
	}
	return fibonacci(n-1) + fibonacci(n-2)
}

func main() {
	result := fibonacci(40)
	fmt.Printf("fib(40) = %d\n", result)
}

File: benchmarks/fibonacci/fib.swift
func fibonacci(n: UInt32) -> UInt32 {
    if n <= 1 {
        return n
    }
    return fibonacci(n: n - 1) + fibonacci(n: n - 2)
}

let result = fibonacci(n: 40)
print("fib(40) = \(result)")
